lib/qmp.mli

```diff
@@ -89,6 +89,8 @@ type message =
   | Event of event
 (** an individual message sent or received to or from qemu *)
 
+val _JSONParsing : string
```

Comment:
I think it was to permit the name and case of the exposed string be the same as the value, to make it easier to spot what the contents are when using it.